pysand/erosion.py

```diff
@@ -296,4 +296,54 @@ def probes(v_m, rho_m, Q_s, D, d_p, alpha=60, K=2e-9, n=2.6, rho_t=7850):
         C2 = 1
     C_unit = 3.15e10  # Conversion factor from m/s to mm/year (4.60)
     E = K * F(a_rad) * v_m ** n / (rho_t * At) * C2 * Q_s / 1000 * C_unit
+    return E
+
+
+def flexible(v_m, rho_m, mu_m, Q_s, mbr, D, d_p):
+    """
+    Particle erosion for flexible pipes with interlock carcass, model reference to DNVGL RP-O501, August 2015
+    :param v_m: Mix velocity [m/s]
+    :param rho_m: Mix density [kg/m3]
+    :param mu_m: Mix viscosity [kg/ms]
+    :param Q_s: Sand production rate [g/s]
+    :param mbr: Minimum Bending Radius in operation [# ID's]
+    :param D: Minimum internal diameter for the interlock carcass [m]
+    :param d_p: Particle diameter [mm]
+    :return: Erosion rate [mm/y]
+    """
+
+    GF = 2
+    E = bend(v_m, rho_m, mu_m, Q_s, mbr, GF, D, d_p)
+    return E
+
+
+def choke_gallery(v_m, rho_m, mu_m, Q_s, GF, D, d_p, R_c, gap, H, K=8.8e-11, n=2.5, rho_t=14600):
+    """
+    Particle erosion for angle style choke gallery, model reference to DNVGL RP-O501, August 2015
+    :param v_m: Upstream mix velocity [m/s]
+    :param rho_m: Mix density [kg/m3]
+    :param mu_m: Mix viscosity [kg/ms]
+    :param Q_s: Sand production rate [g/s]
+    :param GF: Geometry factor [-]
+    :param D: Upstream pipe diameter [m]
+    :param d_p: Particle diameter [mm]
+    :param R_c: Radius of the choke gallery [m]
+    :param gap: Gap between the cage and choke body [m]
+    :param H: Height (effective) of gallery [m]
+    :param K: Material erosion constant, default = 8.8e-11 (CR-37 Tungsten carbide)
+    :param n: Velocity exponent, default = 2.5 (CR-37 Tungsten carbide)
+    :param rho_t: Target material density [kg/m3], default = 14600 (CR-37 Tungsten carbide)
+    :return: Erosion rate [mm/y]
+    """
+
```

Comment:
Should Rc, gap and H be validated? (locally?)

Reply:
Yes I think they should.. I will make changes and push a new commit

Reply:
Fixed, although the new part of the validation function is not tested..

Reply:
I can add the test then, in the next pull request